entry_types/scrolled/package/spec/frontend/AudioPlayerControls-spec.js

```diff
@@ -34,6 +34,23 @@ describe('AudioPlayerControls', () => {
     expect(getByRole('menuitemradio', {name: 'English (CC)'})).not.toBeNull();
   });
 
+  it('renders waveform player controls', () => {
+    const {getByTestId} = renderInEntry(<AudioPlayerControls audioFilePermaId={10}
+                                                             playerState={getInitialPlayerState()}
+                                                             playerActions={getPlayerActions()}
+                                                             configuration={{
+                                                               playerControlVariant: 'waveform'
+                                                             }}/>, {
+      seed: {
+        audioFiles: [{
+          id: 100,
+          permaId: 10
+        }]
+      }
+    });
+    expect(getByTestId('waveform-controls')).not.toBeNull();
+  });
+
```

Comment:
**praise**: Good call testing the feature on `AudioPlayerControls` since that's the place where it is actually used.